convention.md

```diff
@@ -314,7 +314,7 @@ Any other topic is not part of the Homie convention.
 This convention only covers discoverability of devices and its capabilities.
 The aim is to have standardized MQTT topics for all kind of complex scenarios.
 A Homie device may therefore support extensions, defined in separate documents.
-Every extension is identified by a unique ID.
+Every extension is identified by an unique ID.
```

Comment:
https://www.quora.com/Which-is-the-correct-grammar-usage-a-unique-or-an-unique
"a unique" is correct

Reply:
Fixed it^^